source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -212,7 +208,7 @@ of Home Assistant.
 
 Thanks to [@kmdm], as of this release, we also provide a series version tag
 that always points to the latest patch version of that release, in addition
-to all existing tags we already provide.
+to all existing tags, we already provide.
```

Comment:
This comma is actually incorrect, please revert this one.

Reply:
ok